plugins/callback/opentelemetry.py

```diff
@@ -85,7 +85,7 @@
             key: disable_attributes_in_logs
         version_added: 7.1.0
       store_spans_in_file:
-        default: None
+        default:
```

Comment:
**question**: is this the best way to set an empty string as default value? I could also delete the line all together. both works, so this is a cosmestic choice.

WDYT?

Reply:
This sets `None` (the Python value) as the default. If you want an empty string, use:
```suggestion
        default: ""
```
Generally leaving this away so there's actually no default set is better, if you don't want a default.
```suggestion
```

Reply:
I'm ok with no default at all it seems better to me as well.